docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -161,6 +161,16 @@ values. Typical usage does not require you to configure them.
   - Maximum number of metastore data objects per transaction in the Hive
     metastore cache.
   - `1000`
+* - `delta.metastore.store-table-metadata-enabled`
```

Comment:
use `delta.metastore.store-table-metadata.` as the common prefix for these new options